docs/ref/forms/api.txt

```diff
@@ -1226,6 +1226,7 @@ Methods of ``BoundField``
       attributes can be provided by the ``attrs`` argument.
     * ``use_tag``: A boolean which is ``True`` if the label has an ``id``.
       If ``False`` the default template omits the ``<label>`` tag.
+    * ``tag``: An optional string to customize the tag, defaults to ``label``.
```

Comment:
The `use_tag` docs should be adjusted: _"...the default template omits the ``<label>`` tag."_.